code/upaya/Aptfile

```diff
@@ -0,0 +1 @@
+nginx
```

Comment:
Are we currently using this, or do we expect to soon?  If not perhaps we should start with just the simple ruby buildpack?